### ISSUE TITLE
Migrate from Hack Collections to Hack Arrays

### ISSUE THREAD

**Issue by fredemmott** [hhvm/hack-router#10]

**Comment by connordoner** [comment]
Is this something that you ever made progress on, @fredemmott?

**Comment by connordoner** [comment]
t

**Comment by fredemmott** [comment]
No, as hack-http-request-response-interfaces needs reworking to be practical without FIXMES (until very recently, that was blocked on HSL IO), and I'd prefer to minimize the number of BC-breaking releases by doing both at the same time.

**Comment by lexidor** [comment]
See [issue comment on #22](https://github.com/hhvm/hack-router/pull/22#issuecomment-1563364762) and replies for status.
